src/rules/block-opening-brace-space-before/index.js

```diff
@@ -63,3 +75,12 @@ export default function (expectation) {
     }
   }
 }
+
+export function cssStatementIsIgnoredAtRule(statement, options) {
+  return (
+    options &&
+    options.ignoreAtRules &&
+    statement.type === "atrule" &&
+    matchesStringOrRegExp(statement.name, [].concat(options.ignoreAtRules))
```

Comment:
Should be able to reduce this to `matchesStringOrRegExp(statement.name, options.ignoreAtRules)`